cypress/fixtures/constants.ts

```diff
@@ -10,10 +10,18 @@ export const E2E_EMAIL_ADMIN = {
   email: "admin@e2e.gov.sg",
 } as const
 
+export const E2E_EMAIL_ADMIN_2 = {
+  email: "twodmin@e2e.gov.sg",
```

Comment:
Haha i'll leave the existing name as is then!

I was facing issues with the FE flow because adding a collaborator has a secondary modal to go through - I opted to make a new user where the emails are split more explicitly instead!